dotcom-rendering/src/components/StickyLiveblogAskWrapper.importable.tsx

```diff
@@ -1,31 +1,143 @@
 import { palette, space } from '@guardian/source/foundations';
+import { getGutterLiveblog } from '@guardian/support-dotcom-components';
+import type {
+	ModuleData,
+	ModuleDataResponse,
+} from '@guardian/support-dotcom-components/dist/dotcom';
+import type {
+	GutterPayload,
+	GutterProps,
+} from '@guardian/support-dotcom-components/dist/shared/types';
 import type { Tracking } from '@guardian/support-dotcom-components/dist/shared/types/props/shared';
-import { useEffect, useMemo, useState } from 'react';
+import { useEffect, useState } from 'react';
 import { submitComponentEvent } from '../client/ophan/ophan';
 import { shouldHideSupportMessaging } from '../lib/contributions';
 import { useIsSignedIn } from '../lib/useAuthStatus';
 import { useCountryCode } from '../lib/useCountryCode';
 import { usePageViewId } from '../lib/usePageViewId';
+import type { TagType } from '../types/tag';
 import { useConfig } from './ConfigContext';
-import { GutterAsk } from './marketing/gutters/gutterAsk';
-import { props } from './marketing/gutters/utils/storybook';
-import type { ReactComponent } from './marketing/lib/ReactComponent';
-import {
-	addRegionIdAndTrackingParamsToSupportUrl,
-	createClickEventFromTracking,
-	createInsertEventFromTracking,
-} from './marketing/lib/tracking';
-
-// CSS Styling
-// -------------------------------------------
-
-// outer container handles the position and stickiness
+import { GutterAskWrapper } from './marketing/gutters/GutterAsk';
+
+interface GutterLiveblogAskBuilderProps {
+	shouldHideReaderRevenueOnArticle: boolean;
+	sectionId: string | undefined;
+	tags: TagType[];
+	contributionsServiceUrl: string;
+	countryCode: string;
+	pageViewId: string;
+	pageUrl: string;
+}
+
+const GutterLiveblogAskBuilder = ({
+	shouldHideReaderRevenueOnArticle,
+	sectionId,
+	tags,
+	contributionsServiceUrl,
+	countryCode,
+	pageViewId,
+	pageUrl,
+}: GutterLiveblogAskBuilderProps) => {
+	const [supportGutterResponse, setSupportGutterResponse] =
+		useState<ModuleData<GutterProps> | null>(null);
+
+	const { renderingTarget } = useConfig();
+	const isSignedIn = useIsSignedIn();
+
+	// get gutter props
+	useEffect((): void => {
+		if (isUndefined(countryCode) || isSignedIn === 'Pending') {
+			return;
+		}
+		const tagIds = tags.map((tag) => tag.id);
+
+		const hideSupportMessagingForUser =
+			shouldHideSupportMessaging(isSignedIn);
+		if (hideSupportMessagingForUser === 'Pending') {
+			// We don't yet know the user's supporter status
+			return;
+		}
+
+		const payload: GutterPayload = {
+			targeting: {
+				showSupportMessaging:
+					shouldHideSupportMessaging(isSignedIn) === false,
+				countryCode,
+				mvtId: Number(
+					getCookie({ name: 'GU_mvt_id', shouldMemoize: true }),
+				),
+				isSignedIn,
+				tagIds,
+				sectionId,
+			},
+		};
+
+		getGutterLiveblog(contributionsServiceUrl, payload)
+			.then((response: ModuleDataResponse<GutterProps>) => {
+				if (!response.data) {
+					throw new Error(
```

Comment:
an empty response is normal and not an error - it just means we're not showing a message

Reply:
Fixed, thanks!